File: Task1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the string: ");
        String stroka = scanner.next();
        String vowels = "aeioyuAEIOYU";
        String consonants = "bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ";
        int num_vowels = 0;
        int num_consonants = 0;
        
        for (char c : stroka.toCharArray()) {
            if (vowels.indexOf(c) != -1) {
                num_vowels++;
            } 
        }
          for (char c : stroka.toCharArray()) {
            if (consonants.indexOf(c) != -1) {
                num_consonants++;
            } 
        }
        
        if (num_vowels > num_consonants) {
            System.out.println("Yes");
        } else if (num_vowels < num_consonants) {
            System.out.println("No");
        } else {
            System.out.println("Identical");
        }
    }
}

File: task1.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    print("Enter the string: ")
    val stroka = scanner.next()
    val vowels = "aeioyuAEIOYU"
    val consonants = "bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ"
    var num_vowels = 0
    var num_consonants = 0
    for (c in stroka) {
        if (vowels.contains(c)) {
            num_vowels++
        } 
    }
    for (c in stroka) {
        if (consonants.contains(c)) {
            num_consonants++
        } 
    }
    when {
        num_vowels > num_consonants -> println("Yes")
        num_vowels < num_consonants -> println("No")
        else -> println("Identical")
    }
}
